### PR TITLE
2018 - Q1

### PR DESCRIPTION
**Pull request by stevenweaver**

### CODE REVIEW

**Comment by spond** [comment]
Dear @stevenweaver,

Please review and test. The changed to index.html will also need to be ported over to the Secure HIV-TRACE templates to ensure compatibility.

Best,
Sergei

File: index.html [hunk 1]
```diff
@@ -445,15 +425,15 @@ <h4 class="modal-title">Listing nodes in cluster X</h4>
...
+        var lanl_graph_rendered = new hivtrace.clusterNetwork(lanl_graph, lanl_network_container, lanl_network_status_string, lanl_network_warning, lanl_button_bar_prefix, attributes, filter_edges_toggle, null, null, parent_container);
       }
    }
 }
 
 var initialize_cluster_network_graphs = function () {
 
   //Initialize clusternetworkgraph with json url
-  url = "./example.json";
+  url = "./test/SF.json";
```
Comment (stevenweaver) [inline index.html]: Missing in repo

**Comment by spond** [comment]
Dear @stevenweaver,

Could you please revert to `./example.json`?

Best,
Sergei

**Comment by stevenweaver** [comment]
1) The continuous attribute visualization seems to addressed.
2) This is a very cool feature. There is a bug with attributes, however. 

jan-24-2018 13-39-50

**Pull request merged**